libcontainer/nsenter/nsenter_test.go

```diff
@@ -47,7 +47,7 @@ func TestNsenterValidPaths(t *testing.T) {
 	}
 
 	if err := cmd.Start(); err != nil {
-		t.Fatalf("nsenter failed to start %v", err)
+		t.Fatal("nsenter failed to start:", err)
```

Comment:
Why? The previous form seems completely legit.

Reply:
A semicolon was missing. While adding it, I guess I decided to simplify it, that's it.

Reply:
The other reason is, I was using the same form in the subsequent patches, so I added it here, too, for uniformity.

Nevermind though, I re-did this commit so it uses `%v` in this and similar cases, and moved the patch to be the last one.

PTAL @AkihiroSuda